### ISSUE TITLE
Add an option to overwrite existing file

### ISSUE THREAD

**Issue by qyurila** [hilleer/vscode-yaml-plus-json#53]
#### Is your feature request related to a problem? Please describe.

Currently, if there is an existing file which has the same name as target filename, the conversion fails and this notification showed up:

```txt
file already exist: file:///c%3A/foo/bar.yaml
```

Despite this behavior is way better than overwriting silently, the extension does not give any option to overwrite existing file, which may be a preferable choice to manually deleting it sometimes.

#### Describe the solution you'd like

I'd be happy if the 'file already exist' notification comes with a few of option buttons. It might be like:

```txt
file already exist: file:///c%3A/foo/bar.yaml
Do you want to overwrite it?
[Yes] [No] [Always]
```

...and maybe selecting `Always` option would make a new config line to `/.vscode/settings.json` for it.

**Comment by hilleer** [comment]
Hi @qyurila 

Thank you for opening this issue. Sounds reasonable 🙂 Will have a look at it when I can and keep you updated.

**Comment by hilleer** [comment]
Hi @qyurila 

I'm sorry it took this long to (finally) making this happen.

I have released version `1.12.0` with this new feature, enabled via configuration (as proposed here 🤝) - see [README](https://github.com/hilleer/vscode-yaml-plus-json#config) for details.
